### PR TITLE
toml: init at 0.9.4

### PR DESCRIPTION
**Pull request by Twey**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/toml/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "toml";
+  version = "0.9.4";
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/toml/default.nix]: The `name` attribute is added by `buildPython*` and should therefore be removed.

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by FRidh** [comment]
@GrahamcOfBorg build python.pkgs.toml python3.pkgs.toml

**Comment by FRidh** [comment]
```
Ran 0 tests in 0.000s
```
No tests are found. Either the test runner cannot find tests and needs to be patched, or there simply are no tests in which case the tests need to be disabled. Do include a comment explaining why the tests are disabled.

File: pkgs/development/python-modules/toml/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0bdbpbip67wdm6c7xwc6mmbmskyradj4cdxn1iibj4fcx1nbv1lf";
+  };
+
+  meta = with stdenv.lib; {
+    description = "a Python library for parsing and creating TOML";
+    homepage = "https://github.com/uiri/toml";
+    license = licenses.mit;
+    platforms = platforms.all;
```
Comment (FRidh) [inline pkgs/development/python-modules/toml/default.nix]: should be removed

**Comment by Twey** [comment]
Thanks!

There is a test suite but it's not very Nix-friendly.  I've disabled it for the time being.  Maybe we could add separate packages for `tox` and `toml-test` and make a tester that runs it that way.

**Review by FRidh — APPROVED** [review]

**Pull request merged**